validator/utils.py

```diff
@@ -102,72 +105,17 @@ def get_fixed_data(data_dir):
         # other post-processing steps
 
         needs_rewrite = False
```

Comment:
unreferenced boolean "needs_rewrite" Delete please.

Reply:
Is it referred ~line 136?

Reply:
Yeah but that ref is just a test, we deleted the code that might change it. Both chunks should be removed.